subsys/bluetooth/controller/Kconfig

```diff
@@ -50,3 +49,119 @@ config BLECTLR_THREADSAFETY_LOCK_TIMEOUT_MSEC
 		locked.
 
 endmenu
+
+if FLASH
+config SOC_FLASH_NRF_LL_NRFXLIB
+	bool "Nordic BLE controller flash driver"
+	default y
```

Comment:
I guess this will only select the NRFXLIB flash driver if the NRFXLIB LL is selected?

Reply:
Yes. In `bluetooth/controller/Kconfig` is only included if the NRFXLIB LL is selected.